.ui/pgAccount.pgx

```diff
@@ -54,53 +54,8 @@
 				},
 				"title": "pgAccount",
 				"visible": true
```

Comment:
we shouldn't using userprops inside a page

Reply:
done.